catalog/config-schema.json

```
@@ -100,6 +100,10 @@
     "stackVersion": {
       "type": "string",
       "description": "Stack release version"
+    },
+    "packageRoot": {
+      "type": "string",
+      "description": "Package prefix where package files will be uploaded"
```

```suggestion
      "description": "Prefix where package files will be uploaded"
```

"Package prefix" doesn't make sense to me